virt/virt-4-11-release-notes.adoc

```diff
@@ -105,6 +105,15 @@ Deprecated features are included in the current release and supported. However,
 Removed features are not supported in the current release.
 
 //CNV-16317 NMState is no longer part of CNV
+* {VirtProductName} {VirtVersion} removes support for link:https://nmstate.io/[nmstate] including the following objects:
++
+--
+** `NodeNetworkState`
+** `NodeNetworkConfigurationPolicy`
+** `NodeNetworkConfigurationEnactment`
+--
++
+To continue using nmstate features, you must first install the xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[Kubernetes NMState Operator] from the *OperatorHub* in the {product-title} web console, or by using the OpenShift CLI (`oc`), before updating to {VirtProductName} {VirtVersion}. The Kubernetes NMState Operator preserves and supports the existing nmstate configuration.
```

Comment:
If nmstate is no longer supported, do clusters become unsupported if they use this workaround? Or is nmstate still supported, but no longer being provided by CNV?

I think the support implications here should be clearer.

Reply:
As far as I understand, nmstate is no longer being provided by CNV, but still supported by OCP through the Kubernetes NMState Operator.

Reply:
> As far as I understand, nmstate is no longer being provided by CNV, but still supported by OCP through the Kubernetes NMState Operator.

Exactly